terraso_backend/apps/shared_data/views.py

```diff
@@ -66,6 +72,14 @@ def post(self, request, **kwargs):
         return JsonResponse(data_entry.to_dict(), status=201)
 
 
+def invalid_media_types(files):
```

Comment:
issue: please rename this function so it accurately describes purpose. (1) not all of these file types are "media" (2) functions that return a boolean often use the `is_xyz` form. Something like  `is_valid_shared_file_type` would make sense. (Note that you'd have to add a `not` to line 55.)